tools/distribution/src/release/assets/gh_asset.py

```diff
@@ -138,7 +148,6 @@ def validate(self, zip_directory: DirectoryMatcher):
     DatadogObjcXCFrameworkValidator(),
     DatadogCrashReportingXCFrameworkValidator(),
     CrashReporterXCFrameworkValidator(),
-    KronosXCFrameworkValidator(),
```

Comment:
By removing Kronos validator we won't be able to re-build binaries prior to `1.9.0`. It will fail because of finding unexpected `.xcframework` inside `.zip` archive:
```python
dm.assert_number_of_files(expected_count=validated_count)  # assert there are no other files
```